.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npm run test:eslint
```

Comment:
Why are we running eslint as well as lint-staged, which itself is running eslint?

Reply:
fix that